tests/PageObjects/RemoteMic/RemoteMicManageGamePage.ts

```diff
@@ -0,0 +1,30 @@
+import { expect } from '@playwright/experimental-ct-react';
+import { Browser, BrowserContext, Page } from '@playwright/test';
+
+export class RemoteMicManageGamePage {
+  constructor(
+    private page: Page,
+    private context: BrowserContext,
+    private browser: Browser,
+  ) {}
+
+  public get adjustGameLag() {
+    return this.page.getByTestId('game-input-lag');
+  }
+
+  public async increaseGameInputLag() {
+    await this.adjustGameLag.getByTestId('numeric-input-up').click();
+  }
+
+  public async expectGameInputLagToBe(value: string) {
+    await expect(this.adjustGameLag.getByTestId('numeric-input-value')).toContainText(value);
+  }
+
+  public async goToManagePlayer(playerNumber: string) {
+    await this.page.getByText(`Player ${playerNumber}`).click();
```

Comment:
This e.g. `Player 1` is a player name which doesn't necessary have to be `Player {playerNumber}` - instead, the method should allow passing any name, and the `Player` part should be present in its usages